shared/chat/conversations-list/index.desktop.js

```diff
@@ -108,10 +171,16 @@ const Row = shouldUpdate((props: RowProps, nextProps: RowProps) => {
     return true
   }
 
+  // $FlowIssue dunno
   if (props.unreadCount !== nextProps.unreadCount) {
     return true
   }
 
+  // $FlowIssue dunno
+  if (props.rekeyInfo !== nextProps.rekeyInfo) {
```

Comment:
:fire: This was failing flow because the property is misspelled. I think that when tagging a `$FlowIssue` we should include an explanation why to help prevent mistakes like this. It's a bummer that Flow's error output is so cryptic and hard to sift through.